docs/user-guide/application-context.md

```diff
@@ -120,6 +120,58 @@ In the *Enabled Plugins* list, the following buttons are displayed for each exte
 
 <img src="../img/application-context/plugin-config.jpg" class="ms-docimage" style="max-width:300px;"/>
 
-* The **Open plugin configuration documentation** button <img src="../img/button/docu-plugin.jpg" class="ms-docbutton"/> opens the [Plugins Documentation](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins) in another page.
+* The **Open plugin configuration documentation** button <img src="../img/button/docu-plugin.jpg" class="ms-docbutton"/> opens the [Plugins Documentation](https://mapstore.geo-solutions.it/mapstore/docs/api/plugins#plugins) in another page.
 
 
+## Configure Theme
+
+The last wizard steps allows to configure the theme of the context. In [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) the theme can be **default**, **dark** or **custom**.
+
+<img src="../img/application-context/step_four.jpg" class="ms-docimage"/>
+
+### Default Theme
+
+The **default** theme is automatically applied when the editor clicks the **Save** button <img src="../img/button/save_blue_button.jpg" class="ms-docbutton"/> by skipping the *Configure Theme* page. An example of a default context can be the following:
+
+<img src="../img/application-context/default_theme.jpg" class="ms-docimage"/>
+
+### Dark Theme
+
+To activate the **dark** theme, the editor can click on dark in the drop-down menu.
+
+<img src="../img/application-context/dark_theme.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+An example of a dark context can be the following:
+
+<img src="../img/application-context/dark_context.jpg" class="ms-docimage"/>
+
+### Custom Theme
+
+The editor is allows to customize the theme enabling the **Custom Variables**. 
+
+<img src="../img/application-context/custom_variables.jpg" class="ms-docimage" style="max-width:400px;"/>
+
+Once the **Custom Variables** is enable, the editor can choose the color of the different variables by clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> so thet a color picker is displayed to allow the selection of the desire color, as follows:
```

Comment:
Once **Custom Variables** is enabled, the context editor can modify main, primary and secondary colors for both backgrounds and texts (an helper clarifies the UI elements involved for each field in the form). Clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker is displayed to allow the selection of the desire color, as follows: